src/access.rs

```diff
@@ -1,21 +1,132 @@
+use std::ops::{BitAnd, BitOr, Not};
+
 use crate::{
-    AccessError, AddRuleError, AddRulesError, BitFlags, CompatError, CompatResult,
-    HandleAccessError, HandleAccessesError, Ruleset, TailoredCompatLevel, TryCompat, ABI,
+    AccessError, AddRuleError, AddRulesError, CompatError, CompatResult, HandleAccessError,
+    HandleAccessesError, Ruleset, TailoredCompatLevel, TryCompat, ABI,
 };
-use enumflags2::BitFlag;
 
 #[cfg(test)]
-use crate::{make_bitflags, AccessFs, CompatLevel, CompatState, Compatibility};
+use crate::{AccessFs, CompatLevel, CompatState, Compatibility};
+
+#[macro_export]
+macro_rules! make_bitflags {
+    ($bitflag_type:ident :: {$($flag:ident)|*}) => {
+        $bitflag_type::EMPTY $(.union($bitflag_type::$flag))*
+    };
+}
+
+macro_rules! bitflags_type {
+    (
+        $(#[$bitflags_attr:meta])*
+        $vis:vis struct $bitflags_name:ident: $bitflags_type:ty {
+            $(
+                $(#[$flag_attr:meta])*
+                const $flag_name:ident = $flag_val:expr;
+            )*
+        }
+    ) => {
+        $(#[$bitflags_attr])*
+        #[derive(Copy, Clone, Debug, PartialEq, Eq, Default)]
```

Comment:
The current `Debug` derive only shows the raw integer representing the bitflags. If you want I can write some code to also show the names of the flags that are set like is the case before this PR.

Reply:
Yes, that would be useful for the `Debug` implementation.

Reply:
Added